sale_stock_on_hand_popup/models/res_config_settings.py

```diff
@@ -0,0 +1,11 @@
+from odoo import fields, models
+
+
+class ResConfigSettings(models.TransientModel):
+    _inherit = "res.config.settings"
+
+    group_show_transit_location_stock_wizard = fields.Boolean(
+        "Display also transit location lines in sale line stock popup.",
+        implied_group="sale_stock_on_hand_popup.group_show_transit_location_stock_wizard",
+        default=True,
```

Comment:
**issue**: on a a new installation e.g. runboat, users don't actually have this group, this `default=True` is not sufficient for activating the setting by default.

Reply:
Thanks for letting me know, I tried that just for testing purpose, not really needed